examples/hello-world/grifts/routes.go

```diff
@@ -4,11 +4,11 @@ import (
 	"os"
 
 	"github.com/gobuffalo/buffalo/examples/hello-world/actions"
-	. "github.com/markbates/grift/grift"
+	mbgrift "github.com/markbates/grift/grift"
 	"github.com/olekukonko/tablewriter"
```

Comment:
Whoops, just pushed a commit to fix it. 

when I first started this PR . . . I misread something and thought there was a namespace collision. But there's not.